staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -392,3 +397,22 @@ func (le *LeaderElector) Check(maxTolerableExpiredLease time.Duration) error {
 
 	return nil
 }
+
+// setObservedRecord will set a new observedRecord and update observedTime to the current time.
+// Protect critical sections with lock.
+func (le *LeaderElector) setObservedRecord(observedRecord rl.LeaderElectionRecord) {
```

Comment:
nit: use `*rl.LeaderElectionRecord` instead of `rl.LeaderElectionRecord`

Reply:
I fixed, See this commit:
https://github.com/kubernetes/kubernetes/pull/97419/commits/f8c320bed41bd6c3ca8438f9bc36a8a0f4a187c2